core/src/main/kotlin/no/ntnu/beardblaster/view/JoinLobbyScreen.kt

```diff
@@ -0,0 +1,100 @@
+package no.ntnu.beardblaster.view
+
+import com.badlogic.gdx.Gdx
+import com.badlogic.gdx.graphics.Color
+import com.badlogic.gdx.graphics.GL20
+import com.badlogic.gdx.scenes.scene2d.Stage
+import com.badlogic.gdx.scenes.scene2d.ui.Label
+import com.badlogic.gdx.scenes.scene2d.ui.Skin
+import com.badlogic.gdx.scenes.scene2d.ui.Table
+import com.badlogic.gdx.scenes.scene2d.ui.TextButton
+import com.badlogic.gdx.utils.Align
+import com.badlogic.gdx.utils.viewport.FitViewport
+import ktx.actors.onClick
+import ktx.log.debug
+import ktx.log.logger
+import no.ntnu.beardblaster.AbstractScreen
+import no.ntnu.beardblaster.BeardBlasterGame
+import no.ntnu.beardblaster.assets.Assets
+import no.ntnu.beardblaster.worldHeight
+import no.ntnu.beardblaster.worldWidth
+
+private val LOG = logger<JoinLobbyScreen>()
+
+class JoinLobbyScreen(game: BeardBlasterGame) : AbstractScreen(game) {
+    private lateinit var skin: Skin
+    private lateinit var table: Table
+    private lateinit var heading: Label
+
+    private lateinit var btnStartGame: TextButton
+    private lateinit var btnBack: TextButton
+
+    private val joinLobbyStage: Stage by lazy {
+        val result = Stage(FitViewport(worldWidth, worldHeight))
+        Gdx.input.inputProcessor = result
+        result
+    }
+
+    override fun show() {
+        LOG.debug { "JOIN LOBBY Screen" }
+
+        skin = Skin(Assets.assetManager.get(Assets.atlas))
+        table = Table(skin)
+        table.setBounds(0f, 0f, viewport.worldWidth, viewport.worldHeight)
+
+        // Fonts
+        val standardFont = Assets.assetManager.get(Assets.standardFont)
+
+        // Creating buttons
+        val textButtonStyle = TextButton.TextButtonStyle()
+        skin.getDrawable("button_default").also { textButtonStyle.up = it }
+        skin.getDrawable("button_default_hover").also { textButtonStyle.over = it }
+        skin.getDrawable("button_default_pressed").also { textButtonStyle.down = it }
```

Comment:
As a comment, we should really soon plan on setting up either factory-methods for creating these buttons with slightly different styles (e.g., colors: green, red, purple, orange) (we've duplicated the skin / styling so many times already) 😿

Reply:
Agreed! I created an issue for it in #28.